source/includes/concept/_area.md

```diff
@@ -0,0 +1,26 @@
+## Area of Interest
+
+Areas of interest are geographical regions that API users may have a particular interest on, where you can subscribe to deforestation alerts [(GLAD alerts)](https://www.globalforestwatch.org/howto/webinars/webinar-glad-alerts.html) or fire alerts [(VIIRS alerts)](https://data.globalforestwatch.org/datasets/9ebf069428b140d59fc796fef2e1faa8). Areas of interest can be managed using the endpoints provided by the [Areas service](/index-rw.html#areas-v2), while the subscription logic of notifying API users is handled by the [Subscriptions service](/index-rw.html#subscriptions). Your areas of interest can also be managed by accessing your [MyGFW](https://www.globalforestwatch.org/my-gfw/) account on the Global Forest Watch website.
```

Comment:
I'd rephrase it to emphasize that AoI are custom polygons, as opposed to predefined areas/geometries

Reply:
Agree that we implied that the user could only select areas of interest, rephrased emphasizing the user can create its own custom areas of interest :+1: